cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -131,6 +131,9 @@ type VMTServer struct {
 	// Garbage collection (leaked pods) interval config
 	GCIntervalMin int
 
+	// Number of workload controller items the list api call should request for
+	ListQueryItemsCount int
```

Comment:
Would be it better to call this `ItemsPerQuery`, or `ItemsPerListQuery`?

Reply:
> That is my understanding too. But with the current logic, if an rs or rc does not have owner, but has 0 replicas, then we won't be able to cache it. But in fact we do want to cache these controllers. Did I miss anything?

I don't understand the reason to cache a controller which has 0 replicas, in effect no pod associated. I think we don't build a DTO for such controllers. It might be that I am missing something.. :-)

> Would be it better to call this ItemsPerQuery, or ItemsPerListQuery?

Updated to `ItemsPerListQuery`

Reply:
> > That is my understanding too. But with the current logic, if an rs or rc does not have owner, but has 0 replicas, then we won't be able to cache it. But in fact we do want to cache these controllers. Did I miss anything?
> 
> I don't understand the reason to cache a controller which has 0 replicas, in effect no pod associated. I think we don't build a DTO for such controllers. It might be that I am missing something.. :-)
> 
You are right, I forgot that we don't create DTO for controllers with 0 replicas (the same applies to services with no replicas). Then this is not a concern any more.